components/custom-network/theme-colors.tsx

```diff
@@ -18,25 +18,19 @@ const MoreColoursButton = ({label, isVisible, onClick}) => {
   </Button>;
 }
 
-const themeColorsToThemeString = (colors) => {
-  return Object.entries(colors).reduce((prev, entry) => `${prev} ${entry[1]}` , "")
-}
 
-const ThemeStringInput = ({label, text, error, onChange}) => {
+const ThemeStringInput = ({label, updateKey=0, initialText, error, onChange, onBlur, onPaste}) => {
```

Comment:
I did not account for changes onPaste and the inpuf was funky when just using the text field. 

1. Now it uses the initialText only for the input.
2. When changes are typed into the input, they will be propagated on blur
3. Same when a paste event is captured